### DISCUSSION TITLE
OpenVPN  can't assign IPv4 address

### DISCUSSION THREAD

**Discussion by dredkin** [Entware/Entware#880] [Q&A] [answered]
I'm not sure if it's the right place, but.
I have a Coreelec distribution with entware installed. I  installed a package openvpn-openssl and everything was fine in Coreelec19. Recently I updated to Coreelec 20 and now openvpn can't assign IP address. The log shows this error:

```
2022-12-22 20:37:03 us=334222 do_ifconfig, ipv4=0, ipv6=0
2022-12-22 20:37:03 us=334310 WARNING: OpenVPN was configured to add an IPv4 route. However, no IPv4 has been configured for tap0, therefore the route installation may fail or may not work as expected.

```
AFAICS openvpn can't see that the interface is ipv4, though I have a line in my openvpn.conf file:

`proto=tcp4`

And earlier the same  config worked.

If I assign ip manually when openvpn is running, everything works fine:

```
ip addr add 192.168.72.95 dev tap0
route add -net 192.168.72.0 netmask 255.255.255.0 gw 192.168.72.95

```
Ant advice?

**Comment by ryzhovau** [comment, marked as answer]
We do not provide personal support here, sorry.
